### ISSUE TITLE
Remove repositories from poms

### ISSUE THREAD

**Issue by hugoduncan** [pallet/stevedore#19]
It is stated maven best practice to not include repositories in poms. The repository information is only really useful for development.

**Comment by hugoduncan** [comment]
For development, you will need something like the following in your settings.xml

``` xml
<settings>
  <profiles>
    <profile>
      <id>clojure-dev</id>
      <pluginRepositories>
        <pluginRepository>
          <id>mulli.nu</id>
          <url>http://mulli.nu/maven</url>
          <layout>default</layout>
        </pluginRepository>
        <pluginRepository>
          <id>sonatype-releases</id>
          <url>http://oss.sonatype.org/content/repositories/releases</url>
        </pluginRepository>
      </pluginRepositories>
      <repositories>
        <repository>
          <id>clojars</id>
          <name>Clojars</name>
          <snapshots>
            <enabled>true</enabled>
            <updatePolicy>never</updatePolicy>
            <checksumPolicy>fail</checksumPolicy>
          </snapshots>
          <url>http://clojars.org/repo</url>
          <layout>default</layout>
        </repository>
      </repositories>
    </profile>
  </profiles>

  <activeProfiles>
    <activeProfile>clojure-dev</activeProfile>
  </activeProfiles>
</settings>
```

**Comment by hugoduncan** [comment]
Fixed by 4724de14179df7818df32749b4d07635c4ffb497
